pkg/app/topaz.go

```diff
@@ -78,20 +89,19 @@ func (e *Topaz) Start() error {
 
 	// Add registered services to the health service
 	if e.Manager.HealthServer != nil {
+		healthCheck = e.Manager.HealthServer.Server
 		for serviceName := range e.Configuration.APIConfig.Services {
 			e.Manager.HealthServer.SetServiceStatus(serviceName, grpc_health_v1.HealthCheckResponse_SERVING)
+			e.Manager.HealthServer.Server.SetServingStatus("sync", grpc_health_v1.HealthCheckResponse_NOT_SERVING)
```

Comment:
Moving this line outside of the `for` loop would be more efficient.

Reply:
Thanks for the catch! Moved this out of the loop